### DISCUSSION TITLE
Projects Beta: keep navigation within repo

### DISCUSSION THREAD

**Discussion by brooke-hamilton** [community/community#9465] [Projects and Issues]
For projects that are associated with a single repo I'd like to keep the navigation into the beta projects as part of the repo instead of switching context into the project. When you navigate into a beta project from a repo, there is no visible way to navigate to other parts of the same repo. It's a completely different context, but the work described by the project is in context of the repo.

## Steps to reproduce
1. Open my repo. See the tabs for Code, PRs, Discussions, and Projects, etc. 
2. Click on the Projects tab --> click the link to a beta project.

## Expected Result
From the beta project I can navigate to other parts of my repo with a single click, like Code, PRs, Issues, Settings, etc., because I want to work in the context of my repo.

## Actual Result
I have now left the context of my repo and I'm in the context of the beta project, which is only one aspect of my repo. To get back to the repo I have to click back (which could be a lot of back clicks) or re-type the URL to my repo.

For example, I should be able to look at the beta project connected to my repo under the Projects tab, then click to the Issues tab of my repo, and then back to the project, and then look at commit history, without having to change contexts between "projects" and my repo.

The old projects are navigated and displayed as part of the repo (in the context of the repo), which is how they are used by the team.

**Comment by mkungla** [comment]
Collected all related discussions here: https://github.com/github/feedback/discussions/10448
